core/arch/arm/plat-imx/imx_pl310.c

```diff
@@ -34,19 +39,61 @@ void arm_cl2_config(vaddr_t pl310_base)
 
 void arm_cl2_enable(vaddr_t pl310_base)
 {
-	uint32_t val;
+	uint32_t val __maybe_unused;
 
 	/* Enable PL310 ctrl -> only set lsb bit */
 	write32(1, pl310_base + PL310_CTRL);
 
+#ifndef CFG_PL310_SIP_PROTOCOL
 	/* if L2 FLZW enable, enable in L1 */
 	val = read32(pl310_base + PL310_AUX_CTRL);
 	if (val & 1)
-		write_actlr(read_actlr() | (1 << 3));
+		write_actlr(read_actlr() | ACTLR_CA9_WFLZ);
```

Comment:
Minor: what about:
```
if (read32(pl310_base + PL310_AUX_CTRL) & PL310_AUX_CTRL_FLZW)
	pl310_enable_wflz();
```

Reply:
Not really in scope but OK.

Reply:
`pl310_enable_wflz` is conditionally compiled. I think it's simpler to leave as-is.